ui/src/ducks/app/authError.js

```diff
@@ -0,0 +1,29 @@
+//@flow
+export type AuthErrorState = {
+  isAuthError: boolean,
+};
+
+type AuthErrorAction = {
+  type: 'AUTH_ERROR',
+};
+
+const defaultState: AuthErrorState = {
+  isAuthError: false,
+};
+
+export default function reducer(
+  state: AuthErrorState = defaultState,
+  action: AuthErrorAction,
+) {
+  switch (action.type) {
+    case 'AUTH_ERROR': {
+      return { ...state, isAuthError: true };
+    }
+    default:
+      return { ...state };
+  }
+}
```

Comment:
i'm not familiar with the way it work but what i see from the code, there is no way to remove the isAuthError. And we check in `privateRoute.js` if isAuthError is true. Are we sure that if the user relogin the isAuthError is reset ?

Reply:
Hello @gaudiauj ,
It's a very good concern! but thanks to the authentication workflow, the isAuthError should be reset if we re-login again.

Once the user triggers logout action, the page will be redirected to the IDP, either dex or Keycloak. 

Given the page is fully reloaded, the redux store will be re-initiated as well.